templates/Angular2Spa/webpack.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = (env) => {
             rules: [
                 { test: /\.ts$/, include: /ClientApp/, use: ['awesome-typescript-loader?silent=true', 'angular2-template-loader'] },
                 { test: /\.html$/, use: 'html-loader?minimize=false' },
-                { test: /\.css$/, use: ['to-string-loader', 'css-loader'] },
+                { test: /\.css$/, use: [ 'to-string-loader', isDevBuild ? 'css-loader' : 'css-loader?minimize' ] },
```

Comment:
Thank you @SteveSanderson. In truthiness, the differences in spaces also bothered me, until I realized other parts of the templates were different, so I decide to keep it as it was. But now that I'm aware of your preferences for this project I'll make sure to use "no array spaces" as the standard on future PRs.